src/Common/Dialogs/DeleteDialog.tsx

```diff
@@ -88,7 +88,7 @@ export const DeleteDialog: React.FC<DeleteDialogProps> & { Description?: React.F
                             onChange={handleConfirmationTextChange}
                             label={getLabel()}
                             inputWrapClassName="mt-12 w-100"
-                            placeholder={`Type ${deleteConfirmationText} to confirm`}
+                            placeholder="Type to confirm"
```

Comment:
is this intentional to be app-wide change?

Reply:
yes discussed with Utkarsh